run-kiosk-browser.sh

```diff
@@ -2,5 +2,5 @@
 
 URL=$1
 
-kiosk-browser --allowed-save-as-hostname-pattern localhost --allowed-save-as-destination-pattern "/media/**/*" --autoconfigure-print-config ./printing/printer-autoconfigure.json --url ${URL:-http://localhost:3000}
+kiosk-browser --add-file-perm o=http://localhost:*,p=/media/**/*,rw --autoconfigure-print-config ./printing/printer-autoconfigure.json --url ${URL:-http://localhost:3000}
```

Comment:
any reason we can't limit this to `localhost:3000`?

Reply:
Ahh yes you're right, just updated to 3000 and everything seems to work fine.